features/support/fixtures.rb

```diff
@@ -9,7 +9,6 @@ def self.digital_specialists_service
       lot: 'digital-specialists',
       developerLocations: %w[London Scotland],
       developerPriceMax: "200",
-      developerPriceMin: "100",
```

Comment:
Is there any reason not to create a `digital_specialists_service_dos5` fixture as we did for DOS4?

Reply:
Only that I think it'd be a bit messy to have a separate fixture for each framework and yet another place where schemas are defined. I'm happy to create one though if you think there are benefits

Reply:
No, that makes sense. I was wondering if it would cause issues with DOS4 on staging but it seems to work fine so that's all good!